qlib/kernel/Kernel.rs

```diff
@@ -46,79 +46,67 @@ impl HostSpace {
         HyperCall64(HYPERCALL_HLT, 0, 0, 0, 0);
     }
     
-    /* 
+    //Here exists memory leak, vec in shared struct cannot be dealloc
```

Comment:
This is unclear to me. Is the leak caused because of this function or because _of DeallocShareBuf_?

Reply:
The process struct contains vec, initialized in qvisor by pointer in unsafe, thus no one manages these vecs, I only dealloc the process struct but not the addresses pointed by the pointers inside.